integrations/spiral/README.md

```diff
@@ -69,21 +83,26 @@ return [
 ];
 ```
 
-A more complex configuration can be here found:
+> **Warning**
+> When defining the `engines` section, make sure to use the `default` key to set the default engine. This default engine
+> can then be accessed through the `Schranz\Search\SEAL\Engine` class. If you don't explicitly define a default engine,
+> the first engine listed will be used as the default.
```

Comment:
This is not the expected case of the configuration

Reply:
Second part or first one?

Reply:
As I can see in your code `Engine` will be associated with `default` engine if it exists, or with first engine otherwise